### ISSUE TITLE
训练指标问题

### ISSUE THREAD

**Issue by eccentric003** [Gancheekim/ATM-VFI#3]
作者您好，我在第一轮训练结束后对vimeo-90K和ucf101两个数据集进行了测试，发现得到的psnr/ssim指标与文章中的数据几乎一致，现在要进行第二阶段训练，这个阶段也要训练150epoch吗？还有就是第二阶段的数据集X-TRAIN有打包好的数据链接吗？我找到的那个好像不能用，缺少部分文件导致训练不能开始

**Comment by Gancheekim** [comment]
@eccentric003 補充一些訓練細節：
We first warm-up our models for 2k steps and train with a AdamW [21] optimizer with weight decay 1×10−4 on an NVIDIA RTX3090 GPU. All training data are augmented with random flipping, rotation, and time reversal. The training process has four phases: 
1. `Local Motion Pretraining:`
We inactivate the global motion estimation and train our model on Vimeo90K (random cropped to 256×256 patches) with a batch size of 24. The learning rate is varied from 1.5×10−4 to 7.5×10−5 using cosine annealing for 320k steps.
2. `Global Motion Pretraining:`
We freeze the network parameters except for the component responsible for global motion estimation. The remaining of the network is trained on X-TRAIN (random crop to 448×448 patches) with a batch size of 8. The learning rate is varied from 1×10−4 to 5×10−5 using the cosine annealing for 42k steps. 
3. `Joint Motion Training:`
We train the entire network on Vimeo90K during odd-numbered epochs with a batch size of 20 and on X-TRAIN during even-numbered epochs with a batch size of 6. The learning rate is varied from 4×10−5 to 1×10−5 using the cosine annealing for 420k steps.
4. `Perceptual-Oriented Fine-tuning:` 
In this phase, we only fine-tune the residual refinement module to produce more visually appealing results. All of the hyperparameters are same as those of phase (iii) except for that the update steps are set to 100k and λwarp is set to zero.

第二個階段要訓練多少個epoch，可根據你設定的batch size來決定；按照batch size = 8的情況來換算的話，大概要訓練75～80個epoch。

XTRAIN是XVFI的，要去原作者提供的[dropbox](https://www.dropbox.com/scl/fo/88aarlg0v72dm8kvvwppe/AHxNqDye4_VMfqACzZNy5rU?rlkey=a2hgw60sv5prq3uaep2metxcn&e=1&dl=0)下載，按照[他們的做法](https://github.com/JihyongOh/XVFI?tab=readme-ov-file#X4K1000FPS:~:text=After%20downloading%20the%20files%20from%20the%20link%2C%20decompress%20the%20encoded_test.tar.gz%20and%20encoded_train.tar.gz.%20The%20resulting%20.mp4%20files%20can%20be%20decoded%20into%20.png%20files%20via%20running%20mp4_decoding.py.%20Please%20follow%20the%20instruction%20written%20in%20mp4_decoding.py.)處理
